### PR TITLE
enhance test contract with transfer related methods

### PR DESCRIPTION
**Pull request by schmir**
let me export transfer related data as test data.

### CODE REVIEW

**Comment by schmir** [comment]
@berndbohmeier  If you know why I have to declare my own `setAccount` function, I'd be glad for an explanation (though this isn't urgent or critical)

**Comment by berndbohmeier** [comment]
Did you try `getOwner()` and checked who is the owner?
Maybe the constructor of Ownable is not called for some reason. Inside the constructor the owner is set.

File: contracts/TestCurrencyNetwork.sol [hunk 1]
```diff
@@ -40,4 +104,8 @@ contract TestCurrencyNetwork is CurrencyNetwork {
     {
         return _imbalanceGenerated(_value, _balance);
     }
+
+
+
+
```
Comment (berndbohmeier) [inline contracts/TestCurrencyNetwork.sol]: Do we need those empty lines :P

**Comment by berndbohmeier** [comment]
The constructor syntax was changes, maybe we do something wrong there now.

**Comment by schmir** [comment]
I did ask for the owner. It's the one that I expected (the parity dev address filled with ether).
I've opened an issue for the constructor calls. (#168). I have no hope that the constructor calls fix this issue (I already tried it yesterday).

**Pull request merged**